src/test/java/mil/dds/anet/test/resources/AttachmentResourceTest.java

```diff
@@ -75,6 +75,7 @@ public void testCreateAttachment()
     final AttachmentInput testAttachmentInput =
         AttachmentInput.builder().withFileName("testCreateAttachment.jpg").withMimeType(mimeType)
             .withDescription("a test attachment created by testCreateAttachment")
+            .withCaption("testCaption")
```

Comment:
Add an assertion below that the caption of the report attachment matches the attachment input's caption.